### PR TITLE
Replace STATIC_ITEM with PSTRING_ITEM for print count and completed prints

### PR DESCRIPTION
**Pull request by Haxk20**
# Description

While looking at the info menu on my printer the numbers were not even separated from the text.
This as you can imagine does not look very good to an user.

This was an side effect of commit bff0c68 which deleted the previously used definition and replaced
it with STATIC_ITEM which is wrong.

### Benefits

Makes the info menu little prettier.

### Fixes

Fixes commit bff0c68

### CODE REVIEW

File: Marlin/src/lcd/language/language_en.h [hunk 1]
```diff
@@ -506,8 +506,8 @@ namespace Language_en {
   PROGMEM Language_Str MSG_KILL_EXPECTED_PRINTER           = _UxGT("INCORRECT PRINTER");
 
   #if LCD_WIDTH >= 20
-    PROGMEM Language_Str MSG_INFO_PRINT_COUNT              = _UxGT("Print Count");
-    PROGMEM Language_Str MSG_INFO_COMPLETED_PRINTS         = _UxGT("Completed");
+    PROGMEM Language_Str MSG_INFO_PRINT_COUNT              = _UxGT("Print Count: ");
+    PROGMEM Language_Str MSG_INFO_COMPLETED_PRINTS         = _UxGT("Completed: ");
     PROGMEM Language_Str MSG_INFO_PRINT_TIME               = _UxGT("Total Print Time");
     PROGMEM Language_Str MSG_INFO_PRINT_LONGEST            = _UxGT("Longest Job Time");
     PROGMEM Language_Str MSG_INFO_PRINT_FILAMENT           = _UxGT("Extruded Total");
```
Comment (sjasonsmith) [inline Marlin/src/lcd/language/language_en.h]: Do these not appear in the menu? Why do some need colons and some do not?

Comment (sjasonsmith) [reply Marlin/src/lcd/language/language_en.h]: Do some already have colons added in the code which uses the string, rather than here?

**Comment by sjasonsmith** [comment]
Could you share a picture of the menu, so everybody can see how this is displayed?

File: Marlin/src/lcd/language/language_en.h (hunk 1 above)
Comment (Haxk20) [reply Marlin/src/lcd/language/language_en.h]: These ones behave a little different.
Here a new line with ">" and the characters we wish to show is drawn. Thus not needing : from visual perspective.

**Comment by Haxk20** [comment]
@sjasonsmith Sure thing

**Comment by Haxk20** [comment]
Info Menu with this PR
@sjasonsmith

File: Marlin/src/lcd/language/language_en.h (hunk 1 above)
Comment (Haxk20) [reply Marlin/src/lcd/language/language_en.h]: Tho if needed the lines i changed could be changed to behave like the ones you selected here in code.

**Comment by sjasonsmith** [comment]
Thanks! I see now that those other ones use two lines, so the colon is less important.

It is possible that the colons should be added by the calling code, rather than in the language strings. This might make them more usable for different display types which present the data differently. I'm not familiar enough with the various displays and string code to know which would be preferred, but I'm sure @thinkyhead will.

**Comment by Haxk20** [comment]
That is definetly possible. Tho i havent found any recent PR or commit removing such code.

Edit: Actually bff0c68078664fcb229e2b9a9d4eb6b858df2dd2 changes this and is very much likely to be the cause of this issue.
Thus i will force push a better fix.

**Comment by Haxk20** [comment]
@sjasonsmith This is the proper way of fixing this. Please if you want review it. Thank you.

**Comment by Haxk20** [comment]
@thinkyhead should I drop my commit orrrr ? Cause you based the partial revert on top of my commit.
Or do you want to merge it as it is now ?

**Pull request merged**